srcpkgs/audiotube/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'audiotube'
+pkgname=audiotube
+version=22.04
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_PREFIX_PATH=/usr/lib/python3.10/site-packages/pybind11/share/cmake"
```

Comment:
don't hardcode python version here, instead use `${py3_sitelib}`

Reply:
It doesn't do the thing when I use the py3_sitelib variable

Reply:
it isn't going to change any behaviour until py3.11 is released